### DISCUSSION TITLE
Best way to sync Avatars between clients

### DISCUSSION THREAD

**Discussion by Shinnonda** [pixiv/vroid-sdk-developers#37] [Q&A] [answered]
Hi! I've been just learning about the SDK so i can implement into my project, which is a multiplayer game. But i'm having problems to figure how to sync properly the model through clients, since (at least for now) i just figure ways to sync if both have the model liked already, i looked that i could cache it on the game server, and load it on the client, but i'm avoinding that to be more secure and to avoid almost crossing Vroid SDK's Terms of Service and Guidelines.

There's any way to load a model that is not already liked or created for the logged account? it will be just so they can see theirselves on the same ambient.

**Comment by Shinnonda** [comment]
After searching and testing for a while, i saw that Craftopia managed to do it, you can see avatars that are not liked or owned if all players are connected on their respective Vroid Hub accounts, but i don`t know how yet..

**Comment by nyamadan** [comment, marked as answer]
We consider that supporting multiplayer requires more careful handling, as it requires the use of models that are not accessible to the user for rendering purposes.

We ask about single-player implementations and specific uses for the multiplayer SDK and sign an NDA before we give it to you.
Please contact us using the contact form below with an application that includes a single-player SDK that we can review and a specific description of what you plan to use it for.
https://www.pixiv.net/support?mode=inquiry&type=43&service=vroid-sdk

We apologize for the inconvenience, but we appreciate your understanding and cooperation in preventing the unintended use of the 3D models everyone has spent so much time creating.
